app/forms/user_phone_form.rb

```diff
@@ -44,7 +44,8 @@ def ingest_submitted_params(params)
       submitted_phone,
       country_code: international_code
     )
-    self.otp_delivery_preference = params[:otp_delivery_preference]
+    self.otp_delivery_preference = params[:otp_delivery_preference] if
+      params[:otp_delivery_preference]
```

Comment:
What Reek wants you to do here is to assign `params[:otp_delivery_preference]` to a variable and call the variable instead of calling `params[:otp_delivery_preference]` twice.

It's not obvious from Code Climate's website, but there is a help section that explains what the offense is and how to fix it. From the GitHub PR, if you click on "Details" next to the Code Climate check, it will take you to the Code Climate website with a list of all the issues. If you hover your cursor over the issue, a set of 3 icons will appear on the right. The one in the far left looks like a book, and if you hover over it it will say "Read up". If you click on it, it will display a modal explaining what the issue is and how to fix it. Here's a screenshot:
<img width="1126" alt="screen shot 2018-04-30 at 10 40 08 am" src="https://user-images.githubusercontent.com/811150/39433058-f5f2ed86-4c62-11e8-9875-244f48d9b402.png">

Reply:
Something like this @monfresh ?
```
  tfa_prefs = 'params[:otp_delivery_preference]'

    self.otp_delivery_preference = tfa_prefs if
      params[:otp_delivery_preference]

```